tasks/pin_node_group.rb

```diff
@@ -19,10 +23,11 @@
 
 # Get the id of the 'PE Master' node group
```

Comment:
This comment and the one at 29 need to be changed now that these aren't specific to `PE Master`.

Reply:
good call